php.ini-development

```diff
@@ -517,6 +517,15 @@ report_memleaks = On
 ; This setting is on by default.
 ;report_zend_debug = 0
 
+; Set this to disable non-ASCII logging (more secure).
```

Comment:
@bukka The loggers aren't the issue, so much as terminal emulators that allow themselves to be subverted.